### PR TITLE
Adds troubleshooting section to the readme

### PR DESCRIPTION
**Pull request by TMoMoreau**
In response to issue #1162

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: README.md [hunk 1]
```diff
@@ -56,6 +56,40 @@ If you want to build this site locally, run the following:
...
+### Digital envelope routines initialization error
+
+The following error message may display when using Node.js version 18.0.0 when attempting to locally deploy this project for the first time:
+
+   ```shell
+   opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
+   library: 'digital envelope routines',
+   reason: 'unsupported',
+   code: 'ERR_OSSL_EVP_UNSUPPORTED'
+   ```
```
Comment (johnnymatthews) [inline README.md:72]:
```suggestion
   ```

```

File: README.md [hunk 2]
```diff
@@ -56,6 +56,40 @@ If you want to build this site locally, run the following:
...
+2. Navigate into the `ipfs-docs` folder:
+   
+   ```bash
+   cd ipfs-docs
+   ```
+3. Run the following command:
+
+   ```shell
+    export NODE_OPTIONS=--openssl-legacy-provider
+   ```
```
Comment (johnnymatthews) [inline README.md:85]:
```suggestion
   ```

```

File: README.md [hunk 3]
```diff
@@ -56,6 +56,40 @@ If you want to build this site locally, run the following:
...
+4. Run `npm start`.
+
+   ```bash
+   npm start
+   ```
```
Comment (johnnymatthews) [inline README.md:90]:
```suggestion
   ```

```

File: README.md [hunk 4]
```diff
@@ -56,6 +56,40 @@ If you want to build this site locally, run the following:
...
+You can now return to the `Project set up` section above and continue on with the steps. You can also check [this issue](https://github.com/webpack/webpack/issues/14532) for more information about this error.
```
Comment (johnnymatthews) [inline README.md:91]:
```suggestion
You can now return to the [Project set-up](#project-set-up) section above and continue on with the steps. You can also check [this issue in the Webpack GitHub repository](https://github.com/webpack/webpack/issues/14532) for more information about this error.
```

**Review by johnnymatthews — APPROVED** [review]
Nice! Made a couple of tiny errors. Once you're happy feel free to merge.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**